openstef/pipeline/create_component_forecast.py

```diff
@@ -92,9 +102,29 @@ def create_components_forecast_pipeline(
     try:
         input_data = create_input(pj, input_data, weather_data)
 
-        # Save and load the model as .sav file
+        # Save and load the model as .sav file (or as .z file)
         # For the code contact: korte.termijn.prognoses@alliander.com
-        dazls_model: Dazls = joblib.load(DAZLS_STORED)
+        dazls_model = Dazls()
+        dazls_model.domain_model = joblib.load(DAZLS_STORED + "domain_model.z")
+        dazls_model.domain_model_scaler = joblib.load(
+            DAZLS_STORED + "domain_model_scaler.z"
+        )
+        dazls_model.domain_model_input_columns = joblib.load(
+            DAZLS_STORED + "domain_model_features.z"
+        )
+
+        dazls_model.adaptation_model = joblib.load(DAZLS_STORED + "adaptation_model.z")
+        dazls_model.adaptation_model_scaler = joblib.load(
+            DAZLS_STORED + "adaptation_model_scaler.z"
+        )
+        dazls_model.adaptation_model_input_columns = joblib.load(
+            DAZLS_STORED + "adaptation_model_features.z"
+        )
+
+        dazls_model.target_columns = joblib.load(DAZLS_STORED + "target.z")
+        dazls_model.target_scaler = joblib.load(DAZLS_STORED + "target_scaler.z")
+
+        print(dazls_model)
```

Comment:
It is not a good practice to directly print strings. If you think it is beneficial to log certain information, you should use a logger with a static string. You can add key-value pairs to add dynamic information to the log message. E.g.:
```
import structlog

logger = structlog.get_logger(__name__)

logger.info("DAZLS model loaded", dazls_model=str(dazls_model))
```